viewing_party/party.py

```diff
@@ -1,23 +1,128 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+def create_movie(movie_title, genre, rating):
+    new_movie = {}
+    if movie_title == None or genre == None or rating == None:

```

Comment:
Here you are supposed to check if this is truthy but this only checks for None. To check if it's truthy you could so something like `if not title or not genre or not rating`